packages/cli/commands/accounts/clean.js

```diff
@@ -35,7 +42,8 @@ exports.handler = async options => {
 
   trackCommandUsage('accounts-clean', null);
 
-  const filteredTestAccounts = config.portals.filter(p =>
+  const accountsList = getAccounts(config);
```

Comment:
Looks like `getAccounts()` doesn't take in any args anymore

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1177/commits/70875323cba89bb8f18ea7252527a2b6a008e6df